### ISSUE TITLE
[FormDialog] Add ability to set the dialog to full screen in mobile screens

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#315]
In the mobile layout, the dialog should be in full screen with the button row fixed to the bottom of the screen. While setting the dialog to full screen, a state should be added to history to handle the back button of the browser.

**Comment by ramtinabadi** [comment]
The state handling is added, only full screen should be handled
